.ci/install_proxy.sh

```diff
@@ -23,10 +23,23 @@ source "${cidir}/lib.sh"
 
 clone_build_and_install "github.com/clearcontainers/proxy"
 start_proxy_cmd="sudo systemctl start cc-proxy"
+upstart_source_file="${cidir}/data/cc-proxy.conf"
+upstart_destr_file="/etc/init/cc-proxy.conf"
+
+
+# If we are running under the metrics CI system then we do not want the proxy
+# to be dynmaically changing the KSM settings under us - we need control of them
+# ourselves
+if [[ $METRICS_CI ]]; then
+	proxy_systemd_file=$(sudo systemctl show cc-proxy | fgrep FragmentPath | awk 'BEGIN{FS="="}{print $"}')
+	sudo sed -i 's/\(^ExecStart.*$\)/\1 -ksm initial/' "${proxy_systemd_file}"
+
+	upstart_source_file="${cidir}/data/cc-proxy.conf.noksm"
```

Comment:
OOI, why a different file for Upstart but a `sed` to the existing one for systemd?

Reply:
I just modified what existed - I think this is because the systemd file comes from the proxy code/install itself, but the upstart file comes from the .ci tree (where this script lives).
@chavafg I'll let you comment on that if need be